cli.mjs

```diff
@@ -18,16 +19,73 @@ let OUTPUT_ROOT; // The root in which data is saved
 const INVALID_ACTION_ERROR = new Error("Invalid action: " + ACTION);
 const INVALID_DEPLOYMENT_ERROR = new Error("Invalid deployment: " + DEPLOYMENT);
 
+/** -------------------- COMMANDER -------------------- */
+const commander = new Command();
+// default: [download | delete ] not provided, run main() as usual continuing with prompting
+commander.action(() => {});
+
+// download: optional argument studyID and participantID skips relative prompts
+commander
+  .command(`download`)
+  .argument(`[studyID]`)
+  .argument(`[participantID]`)
+  .description(`Download experiment data from Firebase provided study ID and participant ID`)
+  .action((studyID, participantID) => {
+    ACTION = "download";
+    STUDY_ID = studyID;
+    PARTICIPANT_ID = participantID;
+  });
+
+// delete: optional argument studyID and participantID skips relative prompts
+commander
+  .command(`delete`)
+  .argument(`[studyID]`)
+  .argument(`[participantID]`)
+  .description(`Delete experiment data from Firebase provided study ID and participant ID`)
+  .action((studyID, participantID) => {
+    ACTION = "delete";
+    STUDY_ID = studyID;
+    PARTICIPANT_ID = participantID;
+  });
+commander.parse();
+
+// print message if download or delete provided, along with optional args provided
+if (ACTION != undefined) {
+  console.log(
+    `${ACTION} data from Firebase given ${STUDY_ID == undefined ? "" : `study ID: ${STUDY_ID}`} ${PARTICIPANT_ID == undefined ? "" : `and participant ID: ${PARTICIPANT_ID}`}`
+  );
+}
+
 /** -------------------- MAIN -------------------- */
 
 // TODO @brown-ccv #289: Pass CLI arguments with commander (especially for action)
 async function main() {
-  ACTION = await actionPrompt();
+  if (ACTION == undefined) {
+    ACTION = await actionPrompt();
+  }
   DEPLOYMENT = await deploymentPrompt();
   // TODO @brown-ccv #291: Enable downloading all study data at once
-  STUDY_ID = await studyIDPrompt();
+  if (STUDY_ID == undefined) {
+    STUDY_ID = await studyIDPrompt();
+  } else {
+    // when args directly passed in through CLI, check if study is valid
+    const hasStudy = await validateStudyFirebase(STUDY_ID);
+    if (hasStudy != true) {
+      console.error(hasStudy);
+      return;
```

Comment:
What happens on this return? Are there any more prompts or does the program just end?

Reply:
This will return from `main()`, so the program would end. 

Similar to `INVALID_ACTION_ERROR` and `INVALID_DEPLOYMENT_ERROR` (line 19-20), we could create a `INVALID_STUDY_ERROR` / `INVALID_PARTICIPANT_ERROR` to be raised here instead of returning.

Reply:
Let's actually call `process.exit(1)` to indicate that the program ended with an error